org.mixedrealitytoolkit.uxcomponents/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "org.mixedrealitytoolkit.uxcomponents",
-  "version": "3.2.0-development",
```

Comment:
No need to update the minor version. The minor version has already been updated.

Before updating versions, check what version was last released:

https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/releases

Reply:
Updated in latest commit.